web/themes/custom/pineapple/templates/field/field--paragraph--field-p-ig-item.html.twig

```diff
@@ -0,0 +1,64 @@
+{#
```

Comment:
Rename this file to `field--field-p-ig-item.html.twig`

Reply:
done.